pages/[...slug].tsx

```diff
@@ -1,34 +1,46 @@
-import { GetServerSideProps, GetServerSidePropsContext } from 'next';
-import { generateMdx, getDefaultTheme, getPageContent } from '@cmpsr/nextjs-contentful-renderer';
+import { GetStaticPaths, GetStaticProps } from 'next';
 import { Page } from '../src/Page';
+import { staticPaths } from '../src/staticPaths';
+import { getStaticPropsContent } from '../src/getStaticPropsContent';
 
-export const getServerSideProps: GetServerSideProps<any> = async (context: GetServerSidePropsContext) => {
-  const notFound: { notFound: true } = {
-    notFound: true,
+export const getStaticPaths: GetStaticPaths = async () => {
```

Comment:
this changes are wrong, this dynamic route is expected to serve the dynamic pages, not the static ones